server/common/health.py

```diff
@@ -24,10 +24,13 @@ def health_check(config):
     health = {"status": None, "version": "1", "releaseID": cellxgene_version}
 
     checks = False
-    if config.single_dataset__datapath is not None:
-        checks = _is_accessible(config.single_dataset__datapath, config)
-    elif config.multi_dataset__dataroot is not None:
-        checks = all([_is_accessible(datapath, config) for datapath in config.multi_dataset__dataroot.values()])
+    server_config = config.server_config
+    if server_config.single_dataset__datapath is not None:
```

Comment:
these checks (first check single, then check multi) are the opposite of what the comment in default_config.py says (it says that multi is enabled by setting mult_dataset__dataroot).  This code will assume single if single is set, regardless of multil.

Reply:
The code only lets you have multi_dataset__dataroot set OR single_dataset__datapath set.  If you set both it will give a configuration error.  The rest of the code can then assume that only one is set.  But I'll change that to have a top level server param, like you suggested above, then there won't be any logic like this.